Jenkinsfile

```diff
@@ -11,6 +11,23 @@ def config = jobConfig {
 
 
 def job = {
+    // https://github.com/confluentinc/common-tools/blob/master/confluent/config/dev/versions.json
+    def kafkaMuckrakeVersionMap = [
+            "0.10.0": "3.2.x",
+            "0.10.1": "3.2.x",
+            "0.10.2": "3.2.x",
+            "0.11.0": "3.3.x",
+            "1.0": "4.0.x",
+            "1.1": "4.1.x",
+            "2.0": "5.0.x",
+            "2.1": "5.1.x",
+            "2.2": "5.2.x",
+            "2.3": "5.3.x",
+            "trunk": "master",
+            "master": "master"
+
+    ]
```

Comment:
Can we not use the version in `gradle.properties` instead of this map? We can skip the downstream build for anything older than 5.3.x.

Reply:
`gradle.properties` version does not match muckrake branch (5.4.0-ccs-SNAPSHOT vs master ). we need to find out the muckrake branch against which we want to build tests.

Reply:
Can we file a JIRA to rename ccs-kafka branches starting from 5.3.x to match the CP versioning scheme? That would remove the need for this map, right? We should then cross-link to the JIRA from here and from the JIRA to here.

Reply:
Raised JIRA: https://confluentinc.atlassian.net/browse/CPKAFKA-3433